pontoon/contributors/static/css/contributor.css

```diff
@@ -27,3 +27,25 @@ a.avatar:hover .desc {
 a.avatar:hover .desc ~ img {
   opacity: 0.3;
 }
+
+a.avatar .status {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  height: 24px;
+  align-items: center;
+  justify-content: center;
+  gap: 4px;
+  position: absolute;
+  bottom: 3px;
+  border-bottom-left-radius: 6px;
+  border-bottom-right-radius: 6px;
+  color: var(--light-grey-7);
+  background: var(--button-background-2);
+}
+
+a.avatar .status span {
+  font-size: 12px;
+  font-weight: bold;
+  text-align: center;
+}
```

Comment:
Nit: I'd move these rules to the `profile.css` file, because they are specific for the Profile view.